src/utils/plugins/mainnet.ts

```diff
@@ -433,42 +439,42 @@ const plugins: CollateralPlugin[] = [
     rewardToken: [ZERO_ADDRESS],
   },
   // 3.0 new collaterals
-  {
-    symbol: 'crv3Pool',
-    address: collateralAddresses.crv3Pool,
-    decimals: 18,
-    targetUnit: TARGET_UNITS.USD,
-    referenceUnit: 'DAI',
-    collateralToken: 'DAI',
-    description: '',
-    depositContract: '0xaBd7E7a5C846eD497681a590feBED99e7157B6a3',
-    collateralAddress: underlyingCollateralAddresses.DAI,
-    rewardToken: [CVX_ADDRESS[ChainId.Mainnet], CRV_ADDRESS[ChainId.Mainnet]],
-  },
-  {
-    symbol: 'crveUSDFRAXBP',
-    address: collateralAddresses.crveUSDFRAXBP,
-    decimals: 18,
-    targetUnit: TARGET_UNITS.USD,
-    referenceUnit: 'DAI',
-    collateralToken: 'DAI',
-    description: '',
-    depositContract: '0x27F672aAf061cb0b2640a4DFCCBd799cD1a7309A',
-    collateralAddress: underlyingCollateralAddresses.DAI,
-    rewardToken: [CVX_ADDRESS[ChainId.Mainnet], CRV_ADDRESS[ChainId.Mainnet]],
-  },
-  {
-    symbol: 'crvMIM3Pool',
-    address: collateralAddresses.crvMIM3Pool,
-    decimals: 18,
-    targetUnit: TARGET_UNITS.USD,
-    referenceUnit: 'DAI',
-    collateralToken: 'DAI',
-    description: '',
-    depositContract: '0xe8461dB45A7430AA7aB40346E68821284980FdFD',
-    collateralAddress: underlyingCollateralAddresses.DAI,
-    rewardToken: [CVX_ADDRESS[ChainId.Mainnet], CRV_ADDRESS[ChainId.Mainnet]],
-  },
+  // {
```

Comment:
Why are these ones commented?

Reply:
I thought they weren't deployed yet, but looking back at the deployment output I see them being deployed as well. Will update.

Reply:
Updated it, but I'd like at least one other person to verify the addresses.